packages/edit-site/src/components/global-styles/screen-typography-element.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import TypographyPanel from './typography-panel';
+import ScreenHeader from './header';
+
+const elements = {
+	text: {
+		description: __( 'Manage the fonts used on the site.' ),
+		title: __( 'Text' ),
+	},
+	link: {
+		description: __( 'Manage the fonts and typography used on the links.' ),
```

Comment:
I guess these descriptions should be more consistent and change only the part `on the site|links`, no?

Reply:
I'm happy to use any proposed descriptions here :)

Reply:
So I guess we should remove the export here?